src/main/java/Duke.java

```diff
@@ -6,5 +8,98 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greeting();
+    }
+
+    public static void showList(Todo list[], int x){
+        if(x == 0)  System.out.println("\n____________________________________________________________");
+        else{
+            System.out.println("\n____________________________________________________________");
+            for(int i = 0; i < x; i++){
+                if(list[i] instanceof Deadline){
+                    Deadline dummy = (Deadline) list[i];
+                    System.out.println("[" + list[i].returnType() + "] " + "[" + list[i].getStatusIcon() + "] " + list[i].getDescription() + " (" + dummy.getBy() + ")");
+                }
+                else if(list[i] instanceof Event){
+                    Event dummy = (Event)list[i];
+                    System.out.println("[" + list[i].returnType() + "] " + "[" + list[i].getStatusIcon() + "] " + list[i].getDescription() + " (" + dummy.getDuration() + ")");
+                }
+                else{
+                    System.out.println("[" + list[i].getStatusIcon() + "] " + "[" + list[i].returnType() + "] " + list[i].getDescription());
+                }
+            }
+
+            System.out.println("\n____________________________________________________________");
+        }
+    }
+
+    public static void doneTodo(Todo list[], String order){
+        String dummy = order.trim();
```

Comment:
Maybe you could rename "dummy" to another variable like "inputSplitter" so that its purpose is clearer when debugging